src/object-detection/index.rst

```diff
@@ -39,29 +39,22 @@ Below we start DeepStack with only the Detection APIs enabled.
     -p 80:5000 deepquestai/deepstack:gpu
 
 
-*Basic Parameters*
-
-**-e VISION-DETECTION=True** This enables the object detection API, all apis are disabled by default.
-
-**-v localstorage:/datastore** This specifies the local volume where deepstack will store all data.
-
-**-p 80:5000** This makes deepstack accessible via port 80 of the machine.
+Starting DeepStack on other Platforms
+-------------------------------------
 
+`DeepStack on Windows (CPU and GPU) <windows>`_
 
-Starting DeepStack on Raspberry PI
-----------------------------------
+`DeepStack on Raspberry PI <raspberry-pi>`_
 
-.. code-block:: bash
 
-    sudo deepstack start "VISION-DETECTION=True"
+*Basic Parameters*
 
+**-e VISION-DETECTION=True** This enables the object detection API, all apis are disabled by default.
 
-Starting DeepStack on Windows
------------------------------
+**-v localstorage:/datastore** This specifies the local volume where deepstack will store all data.
```

Comment:
This arg is optional right? I believe otherwise docker chooses a secret location for it, but data is still persisted between restarts

Reply:
It is important to specify a Docker volume for use every time to ensure whatever data ( face registrations ) is generated by the container can persist across re-runs and restarts, else the data will be destroyed when the container is destroyed/restarted.

Reply:
Got it, thanks!